src/styles/gradients.css

```diff
@@ -4,3 +4,15 @@
     linear-gradient(to right, #3580c5, #69d974) border-box;
   border: 2px solid transparent;
 }
+
+.join-team-gradient-border {
+  border-right: 1px solid rgba(255, 255, 255, 0);
```

Comment:
Your borders are invisible because your "alpha" is set to zero. Find a way to create a border with a gradient that fades to 0%. You can check the code for the navbar it should be very similar to how the navbar was made.